llvm/test/ExecutionEngine/JITLink/x86-64/LocalDependencyPropagation.s

```diff
@@ -7,8 +7,13 @@
 # symbols: _baz depends on _foo indirectly via the local symbol _bar. We expect
 # _baz to depend on _foo, and _foo on _external_func.
```

Comment:
What about updating this comment with what we expect simplification to do? It wasn't immediately obvious to me, but one of the comments in `simplifyDepGroups()` was useful. Maybe something like "propagate dependencies to symbols outside this unit"?